### PR TITLE
adapt to !5548

### PR DESCRIPTION
**Pull request by shayne-fletcher**
ghc-prim-ops MR https://gitlab.haskell.org/ghc/ghc/-/merge_requests/5548#top

### CODE REVIEW

File: ghc-lib-gen/src/Ghclibgen.hs [hunk 1]
```diff
@@ -224,6 +224,7 @@ compilerDependencies ghcFlavor =
            , "primop-is-cheap.hs-incl"
          ]
   ] ++
+  ["primop-deprecations.hs-incl" | series > GHC_9_10 ] ++
```
Comment (samuel-williams-da) [inline ghc-lib-gen/src/Ghclibgen.hs:227]:
```suggestion
  [ "primop-deprecations.hs-incl" | series > GHC_9_10 ] ++
```

**Review by samuel-williams-da — COMMENTED** [review]
Very minor, then I can run CI

**Comment by shayne-fletcher** [comment]
@samuel-williams-da ping

**Review by dylant-da — APPROVED** [review]

**Pull request merged**